SteamBundleSitesExtension.js

```diff
@@ -842,12 +878,40 @@ const siteHandlers = {
         GM_addStyle(`
             .SBSE_container { margin-top: 10px;}
             .SBSE_container > textarea { border: 1px solid #CC001D;}
-            .SBSE_container button { background-color: #CC001D; color: white;}
+            .SBSE_container button { background-color: #CC001D; color: white; border-radius: 3px;}
+            .SBSE_BtnExport { background-color: #CC001D; color: white; border-radius: 3px;}
+            .SBSE_BtnExport:hover {color: white;}
         `);
 
         // dom source
         const source = location.pathname === '/profile' ? 'div[id*="_sale_"].collapse.in' : document;
 
+        let extractKeys = () => {
```

Comment:
Use `const` if you are not changing variable

Reply:
OK, I'll fix them all